src/Cli/Microsoft.DotNet.Cli.Utils/DotDefaultPathCorrector.cs

```diff
@@ -15,20 +15,17 @@ public static class DotDefaultPathCorrector
         public static void Correct()
         {
             var pathEditor = new WindowsRegistryEnvironmentPathEditor();
-            var dotDefaultPath =
-                pathEditor.Get(
-                    SdkEnvironmentVariableTarget.DotDefault);
+            var dotDefaultPath = pathEditor.Get(SdkEnvironmentVariableTarget.DotDefault);
             if (NeedCorrection(dotDefaultPath, out var correctedPath))
             {
-                pathEditor.Set(correctedPath,
-                    SdkEnvironmentVariableTarget.DotDefault);
+                pathEditor.Set(correctedPath, SdkEnvironmentVariableTarget.DotDefault);
             }
         }
 
-        internal static bool NeedCorrection(string existingPath, out string correctedPath)
+        internal static bool NeedCorrection(string? existingPath, out string correctedPath)
         {
             correctedPath = string.Empty;
-            if (string.IsNullOrWhiteSpace(existingPath))
+            if (existingPath is null || string.IsNullOrWhiteSpace(existingPath))
```

Comment:
Same rationale as explained above with `string.IsNullOrEmpty`. The `string.IsNullOrWhitespace` method has a bit of logic in it so instead of trying to do that here, if we just check for null, it won't cause the nullability analyzer to get upset. Again, just a workaround for .NET Framework's lack of attributes on the method.